__test__/api/comments/updateSingleCommentFlags.test.js

```diff
@@ -0,0 +1,101 @@
+const app = require("../../../server");
+const CommentModel = require("../../../models/comments");
+const commentHandler = require("../../../utils/commentHandler");
+const supertest = require("supertest");
+const request = supertest(app);
+
+describe("PATCH /comments/:commentId/flag", () => {
+  let sampleComment, sampleComment2;
+
+  beforeEach(async () => {
+    const comment = new CommentModel({
+      refId: "94736880",
+      applicationId: global.application._id,
+      ownerId: "myuseremail@email.com",
+      content: "this is a very bad comment",
+      origin: "myuseremail@email.com",
+    });
+
+    const comment2 = new CommentModel({
+      refId: "576897564",
+      applicationId: global.application._id,
+      ownerId: "anotheruseremail@email.com",
+      content: "this is a very bad comment with an non-valid ownerId",
+      origin: "myuseremail@email.com",
+    });
+
+    const commentOne = await comment.save();
+    const commentTwo = await comment2.save();
+
+    sampleComment = commentHandler(commentOne);
+    sampleComment2 = commentHandler(commentTwo);
+  });
+
+  afterEach(async () => {
+    await CommentModel.findByIdAndDelete(sampleComment.commentId);
+    await CommentModel.findByIdAndDelete(sampleComment2.commentId);
+
+    sampleComment = null;
+    sampleComment2 = null;
+  });
+
+  test("Flag a single comment. Should return 200", async () => {
+    const res = await request
+      .patch(`/v1/comments/${sampleComment.commentId}/flag`)
+      .set("Authorization", `bearer ${global.appToken}`)
+      .send({
+        ownerId: sampleComment2.ownerId,
+      });
+    expect(res.status).toBe(200);
+    expect(res.body.status).toEqual("success");
+    expect(res.body.data.numOfFlags).toBeTruthy();
```

Comment:
We no longer encourage the use of `ToBeTruthy` tests, because they are vague and ambiguous and do not give a true picture of what is happening. Instead check for the actual no. of flags. Also make the adjustments in other tests